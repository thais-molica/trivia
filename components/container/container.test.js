import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Container from "./index";

it("Container renders correctly", () => {
  const tree = renderer.create(<Container><div>teste</div></Container>).toJSON();
  expect(tree).toMatchSnapshot();
});
