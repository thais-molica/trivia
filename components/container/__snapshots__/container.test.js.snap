// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly 1`] = `
<div
  className="jsx-3040147036 "
>
  <div>
    teste
  </div>
</div>
`;
